go/engine/revoke.go

```diff
@@ -193,11 +193,15 @@ func (e *RevokeEngine) Run(ctx *Context) error {
 	var sigsList []libkb.JSONPayload
 
 	// Push the per-user-key sig
+
+	// Seqno when the per-user-key will be signed in.
+	var newPukSeqno keybase1.Seqno
 	if e.G().Env.GetSupportPerUserKey() && addingNewPUK {
 		sig1, err := libkb.PerUserKeyProofReverseSigned(me, *newPukSeed, newPukGeneration, sigKey)
 		if err != nil {
 			return err
 		}
+		newPukSeqno = me.GetSigChainLastKnownSeqno()
```

Comment:
Also I just snooped the logs of a run of `TestRevokeDevicePUK` and it looks right.